tests/rules/assertions/checkTagNames.js

```diff
@@ -68,7 +82,67 @@ export default {
                     }
                 }
             }
-        }
+        },
+        {
+            code: `
+                /**
+                 * @yolo foo
```

Comment:
Please use metasyntactic variables for all test cases.

https://en.wikipedia.org/wiki/Metasyntactic_variable#Words_commonly_used_as_metasyntactic_variables